### PR TITLE
De 303 tap twitter invalid time interval

### PR DESCRIPTION
**Pull request by Wizhi**

### CODE REVIEW

**Comment by Wizhi** [comment]
@paddie I seem to have committed the absolute import path in 08a4f6356d41afdc221c221f4428a14b19ad9c76 before we talked about it.

Should I simply remove this commit as not to break anything in prod?

File: tap_twitter_ads/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from twitter_ads.client import Client
 import singer
 from singer import metadata, utils
-from .sync import sync
+from tap_twitter_ads.sync import sync
```
Comment (paddie) [inline tap_twitter_ads/__init__.py:9]: this is fine, just do it consistently across all files (ie. __init__.py etc.)

**Review by paddie — CHANGES_REQUESTED** [review]

File: tap_twitter_ads/__init__.py (hunk 1 above)
Comment (Wizhi) [reply tap_twitter_ads/__init__.py:9]: I've fixed this in 1e5a8b009adc751fe5e35d4c5a03041803480fda.

File: .gitignore [hunk 2]
```diff
@@ -131,6 +131,7 @@ dmypy.json
 # Singer files
 *config.json
 *catalog.json
+*state.json
```
Comment (paddie) [inline .gitignore:134]: 👏

File: tap_twitter_ads/sync.py [hunk 3]
```diff
@@ -1201,8 +1178,7 @@ def sync(client, config, state):
             for platform in platforms_cursor:
                 platform_id = platform["targeting_value"]
                 platform_ids.append(platform_id)
-            LOGGER.info(
-                "Platforms - Platform Targeting IDs: {}".format(platform_ids))
+            LOGGER.info("Platforms - Platform Targeting IDs: {}".format(platform_ids))
```
Comment (paddie) [inline tap_twitter_ads/sync.py:1183]: so proud

**Review by paddie — APPROVED** [review]

**Comment by paddie** [comment]
@Wizhi remember to kick the runner and test it by executing a run for universal_robots

**Pull request merged**